src/promise-katas-1.js

```diff
@@ -1,103 +1,72 @@
-// Example:
-// returnBob() resolves to the value { name: 'Bob' }
-
-const returnBob = () => { };
-
-// 5 Create a function that returns a promise which returns an array with 4 strings of 'eggs', 'apples', 'milk' and 'bread' if it resolves succesfully
-// Example:
-// returnList() resolves to the value ['eggs', 'apples', 'milk', 'bread']
-
-const returnList = () => { };
-
-// 6 Create a function that returns a promise which returns "An error occurred" if it rejects
-// Example:
-// anError() rejects to "An error occurred"
-
-const anError = () => { };
-
-// 7 Create a function that returns a promise which returns 666 if it rejects
-// Example:
-// theNumberOfTheBeast() rejects to the value 666
-
-const theNumberOfTheBeast = () => { };
-
-// 8 Create a function that returns a promise which returns an object with a key of error and value of 500 if it rejects
-// Example:
-// internalServerError() rejects to { error: 500 }
-
-const internalServerError = () => { };
-
-// 9 Create a function that returns a promise which resolves to "happy" if the parameter is greater than
-// or equal to 1 and rejects to "sad" if the value is less than 1
-// Example:
-// happySad(1) resolves to "happy"
-// happySad(0) rejects to "sad"
-
-const happySad = () => { };
-
-// 10 Create a function that returns a promise which resolves to "Yes. Luke, I am your father." if the
-// parameter is "Luke" and rejects to "Not your dad." if the value is anything else
-// Example:
-// amIYourFather() resolves to "Yes. Luke, I am your father."
-// amIYourFather() rejects to "Not your dad."
-
-const amIYourFather = () => { };
-
-// 11 Create a function that returns a promise. The promise should resolve to a new function that returns "My name is <input name>"
-// Example:
-// A call to myNameIs() resolves to a new function that takes one argument, name, and returns "My name is name" Eg (name) => `My name is ${name}`
-
-const myNameIs = () => { };
+const returnPromise = () => {
+  return new Promise((resolve, reject) => {
+    resolve("Good morning!");
+  });
+}
+
+const returnTen = () => {
+  return new Promise((resolve, reject) => {
+    resolve(10);
+  });
+ };
+
+const returnString = () => {
+  return new Promise((resolve, reject) => {
+    resolve("string");
+  });
+ };
+
+const returnBob = () => { 
+  return new Promise((resolve, reject) => {
+    resolve({ name: "Bob" });
+  });
+};
+
+const returnList = () => { 
+  return new Promise((resolve, reject) => {
+    resolve(["eggs", "apples", "milk", "bread"]);
+  });
+};
+
+const anError = () => { 
+  return new Promise((resolve, reject) => {
+    reject("An error occurred");
+  });
+};
+
+const theNumberOfTheBeast = () => { 
+  return new Promise((resolve, reject) => {
+    reject(666);
+  });
+};
+
+const internalServerError = () => { 
+  return new Promise((resolve, reject) => {
+    reject( {error: 500 });
+  });
+};
+
+const happySad = (x) => { 
```

Comment:
I would definitely replace the `x` here with something more meaningful, even `number` would be more helpful as it gives some indication of what is expected to be passed in here. This goes for all other parameters like this in the work